src/vs/workbench/parts/files/browser/fileActions.ts

```diff
@@ -785,6 +786,19 @@ export class ImportFileAction extends BaseFileAction {
 		return this.tree;
 	}
 
+	private bringWindowToForeground(): void {
+		if (isWindows) {
+			try {
+				const { allowSetForegroundWindow } = <any>require.__$__nodeRequire('windows-foreground-love');
+				allowSetForegroundWindow(process.pid);
+			} catch (e) {
+				// noop
+			}
+		}
+
+		this.windowService.focusWindow();
```

Comment:
I suggest to move this closer to the place of the drop operation because that is really where we want to focus the window: https://github.com/Microsoft/vscode/blob/ben/work/src/vs/workbench/parts/files/browser/views/explorerViewer.ts#L735